polaris-ui/src/mock-api/data/searchResults.cypress.ts

```diff
@@ -72,4 +72,26 @@ const searchResults: CaseSearchResult[] = [
       nextHearingDate: "2023-01-02",
     },
   },
+
+  {
+    id: 13401,
+    uniqueReferenceNumber: "12AB2222233",
+    isCaseCharged: true,
+    numberOfDefendants: 2,
+    leadDefendantDetails: {
+      id: 901,
+      listOrder: 0,
+      firstNames: "Steve",
+      surname: "GUZZLERS BREWERY",
+      organisationName: "GUZZLERS BREWERY",
+      dob: "1977-11-28",
+      youth: false,
+      type: "Organisation",
```

Comment:
It would be good if we can find what are the different then we can add union type in that scope , currently it is type is string type.